x-pack/elastic-agent/pkg/agent/operation/monitoring.go

```diff
@@ -168,7 +168,12 @@ func (o *Operator) getMonitoringFilebeatConfig(output interface{}) (map[string]i
 		"filebeat": map[string]interface{}{
 			"inputs": []interface{}{
 				map[string]interface{}{
-					"type":  "log",
+					"type": "log",
+					"multiline": map[string]interface{}{
+						"pattern": `^\d{4}`,
```

Comment:
```suggestion
						"pattern": "^[0-9]{4}",
```
this worked for me, maybe the way how backtick is handled